### ISSUE TITLE
Want to add a Tic Tac Toe Game

### ISSUE THREAD

**Issue by Adit0507** [ZeroOctave/ZeroOctave-Javascript-Projects#189]
**Project Description.**
- A simple tic tac toe game which looks pretty simple but provides a good insight in JS.
- Game can be restarted at any point.
- Live Version: https://adit0507.github.io/Tic-Tac-Toe/

**Appearance**

![Tic-Tac-Toe](https://user-images.githubusercontent.com/78037846/156120689-ec21ec97-c50a-4d6a-95e0-29975a80e80a.PNG)

**Comment by Adit0507** [comment]
I'm a GSSoC''22 participant

**Comment by Parth-Shete** [comment]
I want to contribute to this issue, please let me know what all things I need to keep in my mind before starting contribution.

**Comment by Adit0507** [comment]
I'm almost done with this issue

**Comment by Debdeep1** [comment]
I want to work on this issue under GSSoC'22

**Comment by Astrodevil** [comment]
If issue is already assigned to someone, Find different issue to work on.

**Comment by Astrodevil** [comment]
@Adit0507 If you made new Pr about this. please mention

**Comment by Adit0507** [comment]
No I didn't

**Comment by Astrodevil** [comment]
> No I didn't

Will you do? or I can assign someone else if they are raising this same issue

**Comment by Adit0507** [comment]
> > No I didn't
> 
> Will you do? or I can assign someone else if they are raising this same issue
Isn't this a closed issue ? I already submitted a PR for this

**Comment by Adit0507** [comment]
> > No I didn't
> 
> Will you do? or I can assign someone else if they are raising this same issue

I already submitted a PR for this issue above, and it is closed right now. Why do I need to make a new PR ?

**Comment by SachinMittal1766** [comment]
@Astrodevil I would like to work on it & I also raised a new issue for it but it was closed by mentor. [#345](https://github.com/ZeroOctave/ZeroOctave-Javascript-Projects/issues/345)
